src/Storage/Controllers/InstancesController.cs

```diff
@@ -438,6 +438,7 @@ public async Task<ActionResult<Instance>> Post(string appId, [FromBody] Instance
         [ProducesResponseType(StatusCodes.Status204NoContent)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Comment:
Suggest not specifying that we can return 500, as it's an implicit "sh*t hit the fan" response that most, if not all, endpoints can return.